core-library/src/main/java/org/silverpeas/core/socialnetwork/invitation/AcceptationUserNotification.java

```diff
@@ -0,0 +1,35 @@
+package org.silverpeas.core.socialnetwork.invitation;
+
+import org.silverpeas.core.admin.user.model.UserFull;
+import org.silverpeas.core.notification.user.client.constant.NotifAction;
+import org.silverpeas.core.notification.user.model.NotificationResourceData;
+import org.silverpeas.core.template.SilverpeasTemplate;
+
+/**
+ * Created by Nicolas on 03/02/2017.
+ */
+public class AcceptationUserNotification extends AbstractInvitationUserNotification {
+
+  public AcceptationUserNotification(Invitation invitation) {
+    super(invitation);
+  }
+
+  @Override
+  protected void performTemplateData(final String language, final Invitation resource,
+      final SilverpeasTemplate template) {
+    UserFull user = UserFull.getById(getSender());
+
+    template.setAttribute("senderUser", user);
+    template.setAttribute("userName", user.getDisplayedName());
+  }
+
+  @Override
+  protected void performNotificationResource(final String language, final Invitation resource,
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add a nested comment explaining why this method is empty, throw an UnsupportedOperationException or complete the implementation. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1186)